config/provider.go

```diff
@@ -208,7 +208,6 @@ func GetProvider() *config.Provider {
 		s3.Configure,
 		secretsmanager.Configure,
 		servicecatalog.Configure,
-		devicefarm.Configure,
```

Comment:
I think you removed this statement because it is a duplicate, right?

Reply:
Yes, that's right.